extensions/adobe/experience/offer-management/personalized-offer.schema.json

```diff
@@ -0,0 +1,109 @@
+  "meta:extends": ["https://ns.adobe.com/experience/offer-management/offer"],
+  "type": "object",
+  "title": "Adobe Experience Cloud Personalized Offer",
+  "description": "Adobe Experience Cloud Personalized Offer",
+  "definitions": {
+    "personalized-offer": {
+      "properties": {
+        "@id": {
+          "type": "string",
+          "format": "uri",
+          "title": "ID",
+          "description": "The unique identifier of the offer"
+        },
+        "xdm:name": {
+          "type": "string",
+          "title": "Name",
+          "description": "Offer name"
+        },
+        "xdm:status": {
+          "type": "string",
+          "title": "Status",
+          "description": "Offer Status",
+          "enum": [
+            "draft",
+            "pending",
+            "rejected",
+            "approved",
+            "inUse",
+            "archived"
+          ],
+          "meta:enum": {
+            "draft": "Draft",
+            "pending": "Pending",
+            "rejected": "Rejected",
+            "approved": "Approved",
+            "inUse": "In Use",
+            "archived": "Archived"
+          }
+        },
+        "xdm:representations": {
+          "type": "array",
+          "items": {
+            "$ref":
+              "https://ns.adobe.com/experience/offer-management/offer-representation"
+          },
+          "title": "Representations",
+          "description": "Offer representations for different placements"
+        },
+        "xdm:customMetadata": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string",
+                "title": "Name",
+                "description": "Custom property name"
+              },
+              "value": {
+                "type": "string",
+                "title": "Value",
+                "description": "Custom property value"
+              }
+            },
+            "additionalProperties": false
+          },
+          "title": "Custom Properties",
+          "description":
+            "Custom metadata allows end-users to attach properties to individual instances"
+        },
+        "xdm:rank": {
+          "$ref": "https://ns.adobe.com/experience/offer-management/rank",
+          "title": "Rank",
+          "description": "Offer Rank (priority)"
+        },
+        "xdm:cappingConstraint": {
+          "$ref":
+            "https://ns.adobe.com/experience/offer-management/capping-constraint",
+          "title": "Capping",
+          "description": "Offer capping constraint"
+        },
+        "xdm:selectionConstraint": {
+          "$ref":
+            "https://ns.adobe.com/experience/offer-management/selection-constraint",
+          "title": "Eligibility Rule",
+          "description": "Offer eligibility rule"
+        }
+      },
+      "required": ["xdm:rank", "xdm:cappingConstraint"]
```

Comment:
Why is `xdm:cappingConstraint` required for a personalized offer?

Reply:
Fixed.